### ISSUE TITLE
Underline should appear when focusing on the link with tab key

### ISSUE THREAD

**Issue by alixiashu** [microsoft/fluentui#16842]
### Environment Information

- **Package version(s)**: 7.158.1
- **Browser and OS versions**: Chrome, Win10

### Describe the issue:

Visual users who have low vision and sensitive to colors are getting impacted, as Underline or Chevron is not provided for the 'Breadcrumb' region links, while navigating using keyboard tab key. Users will find difficulty to distinguish between link and plain text.

### Please provide a reproduction of the issue in a codepen:

You could just use the example in [FluentUI/Link](https://developer.microsoft.com/en-us/fluentui#/controls/web/link), I also save a back up in https://codepen.io/alixiashu/pen/yLVeKNa?editors=1010.
Then navigate to the link using tab key to varify whether the underline appears, which should be the same experience as hovering with mouse.

#### Actual behavior:

Underline didn't appear.

#### Expected behavior:

Underline should appear when focusing on the link with tab key.

### Documentation describing expected behavior
[MAS 1.4.1 - Use of Color](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b9f1f0271-42a1-45b1-be00-31473ba2e579%7d)

https://developer.mozilla.org/en-us/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets#general_guidelines

**Comment by joschect** [comment]
@alixiashu Thanks for the submission! While we don't underline the link, I we do provide a box around it to show selection state, please see screenshot. Is the underline needed in addition to the selected state or instead of? I believe that this is intentional on our part overall.

![image](https://user-images.githubusercontent.com/20801176/107443505-d481c580-6aed-11eb-84b9-917238816cff.png)

**Comment by alixiashu** [comment]
hello Jon,
Thanks for the reply. I am a FTE of M365. Actually, it is an accessibility bug that  tester created for our team when we used Link in our page.
I am not sure it is reasonable or not. They need the link to be underlined when it's focused on, to make it differ from common text.
Here is the link Bug 4777478<https://office.visualstudio.com/OC/_workitems/edit/4777478>: [UI fabric][Visual Requirement C M365 Admin Portal C network connectivity - Import ]: Underline/Chevron is not appearing for 'Learn more about office location CSV import' link in 'import' page, while navigating using keyboard tab key.
Please help to take a look, and leave some comment if you have.

best regards,
zhili

________________________________
发件人: Jon Schectman <notifications@github.com>
发送时间: 2021年2月10日 7:45
收件人: microsoft/fluentui <fluentui@noreply.github.com>
抄送: alixiashu <alixiashu@outlook.com>; Mention <mention@noreply.github.com>
主题: Re: [microsoft/fluentui] Underline should appear when focusing on the link with tab key (#16842)


@alixiashu<https://github.com/alixiashu> Thanks for the submission! While we don't underline the link, I we do provide a box around it to show selection state, please see screenshot. Is the underline needed in addition to the selected state or instead of? I believe that this is intentional on our part overall.

[image]<https://user-images.githubusercontent.com/20801176/107443505-d481c580-6aed-11eb-84b9-917238816cff.png>

―

**Comment by joschect** [comment]
@alixiashu Thanks for the extra information! I followed the link and took a look at the video, I do see that the link is not underlined, but I see that it can be interacted with. @smhigley does the underline have to show? Or are our focus boxes enough?

**Comment by smhigley** [comment]
So this one is interesting! The links do need an underline (or some other indication that they are a link), but not when they are focused. The requirement is actually that there is some indication that the links are links vs. regular text when they are not current hovered or focused. That indication could be one of several things:
- an underline (the most common and understood)
- a color that contrasts at least 3:1 from the surrounding text color, in addition to meeting 4.5:1 text contrast against the background
- something else like an icon/border

This comes from the [Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) WCAG criterion. If links are only distinguished from text by color (as they are in this case), then they fail that criterion. Having separate styles on focus/hover isn't enough to pass, since it isn't reasonable to expect a user to move their pointer around the screen to try and discover links (and this wouldn't even work with touch).

Funnily enough, this doesn't matter when the link is focused, because then the focus outline is distinguishing it from the surrounding text 😄

**Comment by joschect** [comment]
@smhigley Thanks for the extra details! @alixiashu I think given the new information and the WCAG info above, our links are working as expected and following the necessary accessibility guidelines.
